src/devices/HCSR501/HCSR501.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Devices.Gpio;
+
+namespace Iot.Device.HCSR501
```

Comment:
Can we rename files, classes and namespaces to use Pascal casing instead? (Hcsr501 instead of HCSR501)

Reply:
I feel like current casing makes more sense but not strong feelings either way